ittapi-rs/ittapi/src/event.rs

```diff
@@ -0,0 +1,75 @@
+use std::{ffi::CString, marker::PhantomData};
+
+/// See the [Event API] documentation.
+///
+/// [Event API]: https://www.intel.com/content/www/us/en/develop/documentation/vtune-help/top/api-support/instrumentation-and-tracing-technology-apis/instrumentation-tracing-technology-api-reference/event-api.html
+pub struct Event(ittapi_sys::__itt_event);
+impl Event {
+    /// Create the event.
+    pub fn new(name: &str) -> Self {
+        #[cfg(unix)]
+        let create_fn = unsafe { ittapi_sys::__itt_event_create_ptr__3_0 };

```

Comment:
what is the __3_0 suffix here .. meaning is this something that will need maintenance with a bump in ittapi versions? Is a string constant or something along those lines useful here and everywhere __3_0 is used?

Reply:
I'm not really sure; this is how the ITT header exposes all of the ITT functions which I assume is for versioning the API. @ekovanova? I can't really use a string; it has to be the linked symbol name (I guess I could hack some macro together to do this but I'm not really sure how often this changes... and if it did, it looks like a simple find and replace).